### DISCUSSION TITLE
Add Users section in the design

### DISCUSSION THREAD

**Discussion by erakastrati** [Budibase/budibase#2389] [Help] [answered]
Hi guys,

Do you know a way to add the Users table(that is generated itself from Budibase) in the development section or it is possible just from the tables part. 
I want to use the Users as Employees of my application without creating a new table named Employees?

Thanks a lot,
Era.

**Comment by kevmodrome** [comment, marked as answer]
I'm a bit uncertain about what you mean here. The Users table is already be available to you in the application. Can you expand a bit on what you mean?

Temporary solution: Add a column to the users table and add some data to each user on that column.

**Reply by erakastrati** [reply]
<img width="1165" alt="Screen Shot 2021-08-17 at 4 30 26 PM" src="https://user-images.githubusercontent.com/74623805/129745140-700e3e3c-248f-48ed-a841-da795724def4.png">
It doesn't appear in the design section

**Reply by kevmodrome** [reply]
Where is it that you expect it to appear? You need to add a Data Provider and select the Users table

**Reply by erakastrati** [reply]
<img width="1674" alt="Screen Shot 2021-08-17 at 4 35 19 PM" src="https://user-images.githubusercontent.com/74623805/129745980-8cf24777-cdce-4a4a-beed-3f035e740d5d.png">
it is not showing my actual users.

**Reply by kevmodrome** [reply]
If you want a new screen called Employees, you first create that screen, then you add a Data Provider, select the Users table. Inside that you select a Table component.

**Reply by kevmodrome** [reply]
That's odd! Can you show me the Data Provider settings?

**Reply by erakastrati** [reply]
<img width="1640" alt="Screen Shot 2021-08-17 at 4 39 08 PM" src="https://user-images.githubusercontent.com/74623805/129746706-7763b69d-4901-4266-b79f-52fe87c50dad.png">
I selected the Users table in the Data Provider.

**Reply by kevmodrome** [reply]
This should definitely be possible, I'll investigate.

**Reply by erakastrati** [reply]
Thanks a lot @kevmodrome .
If possible, text me for any result.

**Reply by erakastrati** [reply]
@kevmodrome have you solved it?

**Reply by kevmodrome** [reply]
Hi Era, no, I have not been able to figure out the issue yet.

**Reply by erakastrati** [reply]
is it only happening in my application, or is there a general problem?

**Reply by kevmodrome** [reply]
No, it seems to be a bug of some sort, I've escalated and created a bug report. Hopefully we'll have it solved soon.

**Reply by erakastrati** [reply]
Thanks a lot for the answer @kevmodrome. I hope you solve it soon.

**Reply by kevmodrome** [reply]
We've identified the problem. In the meantime you can fix it by adding a column to the users table and adding some data to each user.

**Reply by erakastrati** [reply]
thanks @kevmodrome a lot, I tried it and it is working now. 🙂
